### PR TITLE
Correcting a bug with private team members on public projects.

### PR DESCRIPTION
**Pull request by DavidEBest**
Previously, if a person with a private profile was on a public project and the site was built for the public, the private people would be called out as 'Unknown Team Members'. This is corrected, and tests are added to ensure it doesn't happen again.

### CODE REVIEW

**Comment by mtorres253** [comment]
So this just means that if someone has a private profile that person is not shown

**Comment by DavidEBest** [comment]
Yup. Before adding the errors, private profiles were not included. When I added errors, it started adding an error for those private people. Now it behaves the way it used to, except that it'll add errors for actual bogus team members.

**Comment by arowla** [comment]
Does "private profile" mean they are part of the 18F org but haven't made their profile public? I think everybody at 18F is supposed to make their membership public. There might be an opportunity here to help identify private users and ping a Slackbot such as 18f/dolores-landingham-bot to notify the user with instructions on how to go public. (If I'm following this correctly.)

**Comment by mbland** [comment]
@arowla Yes that's the case. You may have a point about making our team members profiles public, but it'd be good to cover this case all the same to avoid surprises.

**Comment by DavidEBest** [comment]
Yup. I'm not sure about what the policy is, but we've got a few folks in the team/private folders that trigger this condition.

**Comment by mtorres253** [comment]
Hi @mbland, can we close this one?

**Comment by mbland** [comment]
Not yet! @DavidEBest can you resolve the conflicts? I can do it if you're swamped. Just wanna give it another peek before merging, then, too.

**Comment by DavidEBest** [comment]
Sure, @mbland, I'll get 'em resolved. Didn't realize there were more. :)

**Comment by mbland** [comment]
@DavidEBest GitHub is still showing conflicts.

**Comment by DavidEBest** [comment]
@mbland. Yup. My git-fu failed me. Checking in again.

File: README.md [hunk 1]
```diff
@@ -42,21 +42,6 @@ Here's a sample bare-bones template you can drop into your prefered layout:
 {% endfor %}
 ```
 
-## Testing
```
Comment (mbland) [inline README.md]: @DavidEBest Make sure these README changes don't get yanked out!

**Comment by DavidEBest** [comment]
:facepalm: Okay, now it should be good.

**Comment by mbland** [comment]
Lovely work, @DavidEBest!

**Pull request merged**